topics/testing/README.md

```diff
@@ -1,6 +1,21 @@
 # Testing
 Before proceeding ensure the tests were built using the [build procedures].
 
+Both unit and integration tests use [Boost.Test](http://www.boost.org/doc/libs/1_62_0/libs/test/doc/html/index.html)
```

Comment:
+1 on the description; however should we also indicate how to launch them in Windows?